src/main/java/school/lemon/changerequest/java/introduction/hw1/Task8.java

```diff
@@ -1,7 +1,14 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
 public class Task8 {
-    public static int calculateSum(int N) {
-        return 0;
+    public static int calculateSum(int N)
+    {
+        int Sum=0;
+        for(int i=0;(N+i)<=2*N;i++)
```

Comment:
Let's say `N = 5`.  So 
`N^2 = 5^2 = 25`. 
`(N+0)^2 = (5 + 0)^2 = 25`
So you can see it's exactly the same thing. Zero is neutral element for addition.